File: 151-ReverseWordsInAString/151-ReverseWordsInAString.java
// Last updated: 5/29/2026, 11:58:05 AM
class Solution {
    public String reverseWords(String s) {
        String[] str=s.trim().split(" ");
        String str1="";
        for(int i=str.length-1;i>=0;i--){
            if(i==0){
                str1+=str[i];
            }
            else if(str[i]==""){
                continue;
            }
            else{
                str1+=str[i]+" ";
            }
        }
        return str1;
    }
}

File: 151-ReverseWordsInAString/151-ReverseWordsInAString.py
# Last updated: 5/29/2026, 11:58:01 AM
class Solution:
    def reverseWords(self, s: str) -> str:
        l=s.split()
        rl=l[::-1]
        o=""
        for i in rl:
            o+=i+" "
        return o.strip()